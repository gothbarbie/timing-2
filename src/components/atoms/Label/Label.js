import styled from 'styled-components'

const Label = styled.label`
  font-size: 1.4rem;
  font-weight: 500;
  color: ${({ theme }) => theme.colors.dark};
`

export default Label
