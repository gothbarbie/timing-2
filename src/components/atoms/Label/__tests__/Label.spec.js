import React from 'react'
import { shallow } from 'enzyme'

import Label from '../Label'

describe('components/atoms/Label', () => {
  it('renders', () => {
    const component = shallow(<Label>Text</Label>)

    expect(component).toMatchSnapshot()
  })
})
