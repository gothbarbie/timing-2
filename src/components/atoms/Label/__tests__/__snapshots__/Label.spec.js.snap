// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/Label renders 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "Label-a0jzeo-0",
        "isStatic": false,
        "rules": Array [
          "font-size:1.4rem;font-weight:500;color:",
          [Function],
          ";",
        ],
      },
      "displayName": "Label",
      "render": [Function],
      "styledComponentId": "Label-a0jzeo-0",
      "target": "label",
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Text
</StyledComponent>
`;
